src/Makefile.am

```diff
@@ -30,6 +30,10 @@ ynclude_DATA = \
   include/dns-server/cmdline.rb \
   include/dns-server/helps.rb
 
+ylibdir = @ylibdir@/ui
```

Comment:
Well, this is not supposed to live in this package for long. Once the prototype is validated, this will be moved to yast-yast2 and it should live next to UI::Dialog and UI::EventDispatcher, unless you see a better place for it.